keyboards/bredworks/wyvern_hs/readme.md

```diff
@@ -0,0 +1,22 @@
+# Bred Works Wyvern HS
+
+* Keyboard Maintainer: [DeskDaily](https://github.com/DeskDaily)
+* Hardware Supported: Atmega32u4
```

Comment:
```suggestion
* Hardware Supported: Atmega32u4
* Hardware Availability: [BredWorks](https://bredworks.com/products/wyvern-hotswap-pcb)
```

A link to the vendor's website is a good way to verify that the firmware is compatible with the hardware (unless there's a good reason for omitting this bullet point from the template).

Reply:
Thanks for the link! I did not know that there is already a product link for the pcb. I was only asked to make the firmware and execute the pull request